jenkins/ci.suse.de/templates/cloud-socok8s-ansible-lint.yaml

```diff
@@ -0,0 +1,20 @@
+- job-template:
+    name: '{name}-ansible-lint'
+    project-type: multibranch
+    periodic-folder-trigger: 5m
+    number-to-keep: 30
+    days-to-keep: 30
+    script-path: Jenkinsfile.lint
+    scm:
+      - github:
+          repo: '{repo-name}'
+          repo-owner: '{repo-owner}'
+          credentials-id: '{repo-credentials}'
+          branch-discovery: no-pr
+          discover-pr-forks-strategy: merge-current
+          discover-pr-forks-trust: permission
+          discover-pr-origin: merge-current
+          submodule:
+            recursive: true
+          notification-context: continuous-integration/jenkins
```

Comment:
not sure, but shouldn't that be something like `continuous-integration/jenkins/lint` ?

Reply:
ah indeed!